fastlane/lib/fastlane/actions/git_commit.rb

```diff
@@ -8,7 +8,9 @@ def self.run(params)
           paths = params[:path].map(&:shellescape).join(' ')
         end
 
-        result = Actions.sh("git commit -m #{params[:message].shellescape} #{paths}")
+        skip_git_hooks = params[:skip_git_hooks] ? '--no-verify' : nil
```

Comment:
```suggestion
        skip_git_hooks = params[:skip_git_hooks] ? '--no-verify' : ''
```
Can we make this an empty string inside of nil?